.github/workflows/ci.yml

```
@@ -0,0 +1,401 @@
+        
+    - name: Build CLI
+      run: make build
+      
+    - name: Test help command response time budget
+      run: |
+        # Help command should respond within 100ms
+        START=$(date +%s%N)
+        ./bin/claude-pilot --help > /dev/null
+        END=$(date +%s%N)
+        DURATION=$((($END - $START) / 1000000))
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)